src/lib/output/renderer.ts

```diff
@@ -282,6 +285,21 @@ export class Renderer extends ChildableComponent<Application, RendererComponent>
         return Path.dirname(require.resolve('typedoc-default-themes'));
     }
 
+    /**
+     * Return the path to the target installed theme.
+     *
+     * @returns The path to the default theme.
+     */
+    static getInstalledTheme(themeName: string): string {
+        if (/[\/\\]/.test(themeName)) {
```

Comment:
what exactly is this regex looking for?

Reply:
@aciccarello for chk theamname not a path, make sure it is name only